### DISCUSSION TITLE
when will version 1 be out?

### DISCUSSION THREAD

**Discussion by globalflea** [serverlessworkflow/specification#526] [Q&A] [answered]
Just wondering what's the timeline for version 1?

**Comment by tsurdilo** [comment]
Here is the latest roadmap timeline that we presented at KubeCon NA recently. 
https://youtu.be/aImJRh8CHFQ?t=1131

**Reply by globalflea** [reply]
Noticed in the video you have a version of the spec running on temporal.io written in java.  is there an equivalent in Go? Of has any work been done to build the entire spec on top of temporal?

**Reply by tsurdilo** [reply]
There is work in progress for integration with Temporal (as you noticed via Java SDKs). We will make the repo public as soon as some more progress has been made.

Not for Go currently. Temporal Java SDK provides DynamicWorkflow interface (https://github.com/temporalio/sdk-java/blob/master/temporal-sdk/src/main/java/io/temporal/workflow/DynamicWorkflow.java) which makes perfect fit for DSL integration. We would need similar construct in other Temporal SDKs first , in order for similar integration to be added.

**Reply by gerbyzation** [reply]
Where did things land with the temporal integration? Sounds really interesting

**Comment by gmuellercole** [comment]
Looking for an update on the 1.0 timeline?

**Reply by tsurdilo** [reply]
We will release 0.9 next month (had some delays due to PTO etc) and then will focus on only bug fixes, no new features until 1.0 (which will be the release version after 0.9).
Don't have a concrete time line as of yet but my guess would be end of July right now. SDKs can have different release dates to update to this version, we will know more specifics when we get there.

**Comment by ricardozanini** [comment, marked as answer]
Versions release can be found on the releases page: https://github.com/serverlessworkflow/specification/releases

So far, we have 1.0.0-alpha1 and 1.0.0-alpha2 releases. You can follow the milestone 1.0.0 release here: https://github.com/serverlessworkflow/specification/milestone/9
